components/pages/installation.html

```diff
@@ -0,0 +1,75 @@
+    </script>
+</head>
+
+<body>
+
+
+    <!-- nav bar -->
+    <nav class="navigation container">
+        <div class="nav-brand">
+            <a href="../../index.html" class="icon"><i class="fas fa-atom"></i></a>
+            Atom UI
+        </div>
+
+        <ul class="list-non-bullet nav-list">
+            <li class="list-item-inline">
+                <a class="link" href="../../index.html">Home</a>
+            </li>
+            <li class="list-item-inline">
+                <a class="link link-active" href="installation.html">Documentation</a>
+            </li>
+            <li class="list-item-inline">
+                <a class="link" href="https://github.com/sharath-io/Atom-UI">Github</a>
+            </li>
+        </ul>
+    </nav>
+
+    <!-- sidebar -->
+    <aside class="sidebar">
+        <ul>
+            <li id="install"><a href="installation.html">Installation</a></li>
+            <li><a href="avatar.html">Avatar</a></li>
+            <li><a href="badge.html">Badge</a></li>
+            <li><a href="alert.html">Alert</a></li>
+            <li><a href="button.html">Button</a></li>
+            <li><a href="card.html">Card</a></li>
+            <li><a href="image.html">Image</a></li>
+            <li><a href="input.html">Input</a></li>
+            <li><a href="navigation.html">Navigation</a></li>
+            <li><a href="rating.html">Rating</a></li>
+            <li><a href="slider.html">Slider</a></li>
+        </ul>
+    </aside>
+
+    <div class="body-text">
+        <h1>Installation</h1>
+        <p class="description">Atom UI is a simple ready to use CSS component library for your next project</p>
+        <p class="description">Paste the below code in head tag of your html code. And you are ready to use Atom
+            UI.</p>
+        <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%2815%2C118%2C110%2C1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Clink%2520href%253D%2522https%253A%252F%252Fatom-ui.netlify.app%252Fcss%252Fmain.css%2522%2520rel%253D%2522stylesheet%2522%253E"
+            style="width: 749px; height: 204px; border:0; transform: scale(1); overflow:hidden;"
+            sandbox="allow-scripts allow-same-origin">
+        </iframe>
+
+        <p class="description">Or can add below code in you main css file.</p>
+        <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%2815%2C118%2C110%2C1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2540import%2520url%28%2522https%253A%252F%252Fatom-ui.netlify.app%252Fcss%252Fmain.css%2522%2520rel%253D%2522stylesheet%2522%29%253B"
+            style="width: 749px; height: 204px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
avoid inline styling, if possible create a class for which will contain border, transform & overflow .. and just keep height in inline as carbon.sh creates issues ..jst a suggestion

Reply:
Thank you for suggestion. Updated code
Created  separate class for iframe but kept width & height inline as it is not same for all iframes in my code.